plugin/trino-openlineage/pom.xml

```diff
@@ -123,6 +123,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <scope>runtime</scope>
```

Comment:
Typically runtime dependencies are not needed. So ideally there could be a code comment saying why this one is needed. BTW the fact we need to manually add this is an effect of incorrect exclude somewhere else?